core/store/migrate/migrate_test.go

```diff
@@ -0,0 +1,265 @@
+					monitoring_endpoint, created_at, updated_at)
+			VALUES (:id, :contract_id, :relay, :relay_config, :p2p_bootstrap_peers, :ocr_key_bundle_id, :transmitter_id,
+					 :blockchain_timeout, :contract_config_tracker_poll_interval, :contract_config_confirmations, :juels_per_fee_coin_pipeline, :is_bootstrap_peer,
+					:monitoring_endpoint, NOW(), NOW())
+			RETURNING id;`
+	_, err = db.NamedExec(sql, jb.Offchainreporting2OracleSpec)
+	require.NoError(t, err)
+	_, err = db.NamedExec(sql, nonBootstrapJob.Offchainreporting2OracleSpec)
+	require.NoError(t, err)
+	_, err = db.NamedExec(sql, jb2.Offchainreporting2OracleSpec)
+	require.NoError(t, err)
+
+	sql = `INSERT INTO jobs (id, pipeline_spec_id, external_job_id, schema_version, type, offchainreporting2_oracle_spec_id, bootstrap_spec_id, created_at)
+		VALUES (:id, :pipeline_spec_id, :external_job_id, :schema_version, :type, :offchainreporting2_oracle_spec_id, :bootstrap_spec_id, NOW())
+		RETURNING *;`
+	_, err = db.NamedExec(sql, jb)
+	require.NoError(t, err)
+	_, err = db.NamedExec(sql, nonBootstrapJob)
+	require.NoError(t, err)
+	_, err = db.NamedExec(sql, jb2)
+	require.NoError(t, err)
+
+	// Migrate up
+	err = goose.UpByOne(db.DB, "migrations")
+	require.NoError(t, err)
+
+	var bootstrapSpecs []job.BootstrapSpec
+	sql = `SELECT * FROM bootstrap_specs;`
+	err = db.Select(&bootstrapSpecs, sql)
+	require.NoError(t, err)
+	require.Len(t, bootstrapSpecs, 2)
+	fmt.Printf("bootstrap count %d\n", len(bootstrapSpecs))
+	for _, bootstrapSpec := range bootstrapSpecs {
+		fmt.Printf("bootstrap id: %d\n", bootstrapSpec.ID)
+	}
+
+	var jobs []job.Job
+	jobs, count, err := jobORM.FindJobs(0, 1000)
+	require.NoError(t, err)
+	require.Equal(t, 3, count)
+	fmt.Printf("jobs count %d\n", len(jobs))
+	for _, jb := range jobs {
+		fmt.Printf("job id: %d with BootstrapSpecID: %d\n", jb.ID, jb.BootstrapSpecID)
+	}
+	require.Nil(t, jobs[1].BootstrapSpecID)
+
+	migratedJob := jobs[2]
+	require.Nil(t, migratedJob.Offchainreporting2OracleSpecID)
+	require.NotNil(t, migratedJob.BootstrapSpecID)
+	require.Equal(t, &job.BootstrapSpec{
+		ID:                                1,
+		ContractID:                        spec.ContractID,
+		Relay:                             spec.Relay,
+		RelayConfig:                       spec.RelayConfig,
+		MonitoringEndpoint:                spec.MonitoringEndpoint,
+		BlockchainTimeout:                 spec.BlockchainTimeout,
+		ContractConfigTrackerPollInterval: spec.ContractConfigTrackerPollInterval,
+		ContractConfigConfirmations:       spec.ContractConfigConfirmations,
+		CreatedAt:                         migratedJob.BootstrapSpec.CreatedAt,
+		UpdatedAt:                         migratedJob.BootstrapSpec.UpdatedAt,
+	}, migratedJob.BootstrapSpec)
+	require.Equal(t, job.Bootstrap, migratedJob.Type)
+
+	sql = `SELECT COUNT(*) FROM offchainreporting2_oracle_specs;`
+	err = db.Get(&count, sql)
+	require.NoError(t, err)
+	require.Equal(t, 1, count)
+
```

Comment:
we should assert that job ID 10's bootstrap_spec does still point to ContractID terra_187246hr3781h9fd198fh391g8f924 and ditto for job 20 and sol_XX